From f84b7eda91d997c2828b9fa7b7ba77db5a4fa680 Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Thu, 1 Jun 2017 21:03:37 +0000
Subject: [PATCH] OS-6159 root.password file contains the hash, not the
 password Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved by:
 Cody Peter Mello <cody.mello@joyent.com>

---
 tools/build_live | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/build_live b/tools/build_live
index 97bf6ee9..cca886a8 100755
--- a/tools/build_live
+++ b/tools/build_live
@@ -375,7 +375,7 @@ function bi_create_password
 	# Stash the generated root password as a plain text build artefact
 	# so that we can find it later if we need it.
 	#
-	if ! printf '%s\n' "$cp_hash" >"$stash"; then
+	if ! printf '%s\n' "$cp_password" >"$stash"; then
 		fail "unable to stash root password in \"$stash\""
 	fi
 
-- 
2.21.0

